demos/python_demos/instance_segmentation_demo/README.md

```diff
@@ -1,6 +1,6 @@
 # Instance Segmentation Python* Demo
 
-This demo demonstrates how to run Instance Segmentation models from `Detectron` or `maskrcnn-benchmark` using OpenVINO&trade;.
+This demo shows how to run Instance Segmentation models from `Detectron` or `maskrcnn-benchmark` using OpenVINO&trade;.
```

Comment:
`Instance Segmentation models` -->`Instance-Segmentation models`

Reply:
Disagree. It's "Instance Segmentation".